weatherman/results.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+
+
+class YearlyExtremeWeatherResult:
+    """
```

Comment:
There should only be single class `WeathermanReportGenerator` for generating yearly or monthly reports

Reply:
I've defined that `WeathermanReportGenerator` in `reportgenerator` module which basically has different methods to generate different weather reports. and those different methods basically uses classes defined in this results module. these classes are basically data structures defined for holding weather results after processing and `reportgenerator `module uses these results to produce reports